src/components/filter/expert/expert-filter-utils.ts

```diff
@@ -239,6 +245,7 @@ export function exportExpertRules(query: RuleGroupType): RuleGroupTypeExport {
 
         // a composite rule is a rule with dataType COMBINATOR  => build a group with child rules
         if (dataType === DataType.COMBINATOR) {
+            // eslint-disable-next-line @typescript-eslint/no-use-before-define
```

Comment:
This one could be fixed no ?

Reply:
no, `transformRule` is using `transformCompositeRule` which is using `transformRule`